### ISSUE TITLE
lost zone states for all except zone one with update to version 1.0.0

### ISSUE THREAD

**Issue by gdlesage** [sebr/bhyve-home-assistant#21]
First great work

updated to version 1.0.0 and saw all the new features which are great, but I lost the zone state sensor for all zones with the exception of zone 1.  I uninstalled, removed all bhyve entities, reinstalled with same results.  My debug file suggests zones 2,3 and 4 states are recognized and are activated (below) but still no zone states show in entities list or in lovelace.  Strange that zone 1 (backyard) does show.  Any thoughts?

Log-->

2020-06-24 13:59:43 INFO (MainThread) [homeassistant.setup] Setup of domain camera took 1.9 seconds.
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.sensor] Creating battery sensor: Smart Outdoor Timer battery level
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.sensor] Creating state sensor: Back Yard zone state
2020-06-24 13:59:43 DEBUG (MainThread) [custom_components.bhyve.sensor] State sensor Back Yard zone state setup: State: auto | Available: True
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.sensor] Creating state sensor: Plants zone state
2020-06-24 13:59:43 DEBUG (MainThread) [custom_components.bhyve.sensor] State sensor Plants zone state setup: State: auto | Available: True
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.sensor] Creating state sensor: Front Right zone state
2020-06-24 13:59:43 DEBUG (MainThread) [custom_components.bhyve.sensor] State sensor Front Right zone state setup: State: auto | Available: True
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.sensor] Creating state sensor: Front Left zone state
2020-06-24 13:59:43 DEBUG (MainThread) [custom_components.bhyve.sensor] State sensor Front Left zone state setup: State: auto | Available: True
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Back Yard zone
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Back Yard zone watering_status: None
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Back Yard Zone: Watering program Half (b) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Back Yard Zone: Watering program Full (a) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Back Yard Zone: Watering program Quarter (c) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Back Yard Zone: Watering program One (d) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Back Yard Zone: Watering program Smart Watering (e) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Smart Outdoor Timer rain delay
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Plants zone
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Plants zone watering_status: None
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Plants Zone: Watering program Half (b) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Plants Zone: Watering program Full (a) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Plants Zone: Watering program Quarter (c) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Plants Zone: Watering program One (d) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Plants Zone: Watering program Smart Watering (e) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Smart Outdoor Timer rain delay
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Front Right zone
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Right zone watering_status: None
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Right Zone: Watering program Half (b) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Right Zone: Watering program Full (a) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Right Zone: Watering program Quarter (c) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Right Zone: Watering program One (d) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Smart Outdoor Timer rain delay
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Front Left zone
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Left zone watering_status: None
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Left Zone: Watering program Half (b) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Left Zone: Watering program Full (a) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Left Zone: Watering program Quarter (c) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Front Left Zone: Watering program One (d) is not enabled, skipping
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Smart Outdoor Timer rain delay
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Program Default
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Program Half
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Program Full
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Program Quarter
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Program One
2020-06-24 13:59:43 INFO (MainThread) [custom_components.bhyve.switch] Creating switch: Program Smart Watering
2020-06-24 13:59:43 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=bhyve, service=enable_rain_delay>
2020-06-24 13:59:43 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=bhyve, service=disable_rain_delay>
2020-06-24 13:59:43 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=bhyve, service=start_watering>
2020-06-24 13:59:43 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=bhyve, service=stop_watering>
2020-06-24 13:59:43 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event service_registered[L]: domain=bhyve, service=set_manual_preset_runtime>

**Comment by sebr** [comment]
@gdlesage Thanks for the report - do you think you could also send the debug output for the devices json dump which should be written to the log with debug enabled?

Thanks!

**Comment by CrouchingWorm** [comment]
I'm having a similar problem.
Thanks so much for this integration!

**Comment by gdlesage** [comment]
not sure what you mean by "devices json", but searching log shows many errors related to apparent missing JSON files,  1220 examples found, most are duplicates with some shown below.   Give me a little more info on what you need and Ill make the search
    On Wednesday, June 24, 2020, 05:28:30 PM EDT, Seb Ruiz <notifications@github.com> wrote:  
 
 


@gdlesage Thanks for the report - do you think you could also send the debug output for the devices json dump which should be written to the log with debug enabled?

Thanks!

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub, or unsubscribe.



some searched json debugs:
2020-06-24 13:59:43 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/map/translations/en.json2020-06-24 13:59:43 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/homeassistant/translations/scene.en.json2020-06-24 13:59:43 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/mobile_app/translations/binary_sensor.en.json2020-06-24 13:59:43 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=default_config>2020-06-24 13:59:43 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/search/translations/en.json2020-06-24 13:59:43 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/group/translations/light.en.json2020-06-24 13:59:43 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/logger/translations/en.json

2020-06-24 13:59:45 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/websocket_api/translations/en.json2020-06-24 13:59:45 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/stream/translations/en.json2020-06-24 13:59:45 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ssdp/translations/en.json2020-06-24 13:59:45 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/alexa/translations/en.json2020-06-24 13:59:45 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ring/translations/camera.en.json2020-06-24 13:59:45 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/nx584/translations/binary_sensor.en.json

2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /config/custom_components/bhyve/translations/sensor.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/met/translations/weather.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/honeywell/translations/climate.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ipp/translations/sensor.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/nx584/translations/alarm_control_panel.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/updater/translations/binary_sensor.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/webhook/translations/en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/logbook/translations/en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ffmpeg/translations/en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/template/translations/switch.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/frontend/translations/en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/history/translations/en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/cloud/translations/en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/system_log/translations/en.json
2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/zeroconf/translations/en.json2020-06-24 13:59:46 DEBUG (SyncWorker_15) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/117426840.hacs2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ring/translations/sensor.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ring/translations/switch.en.json2020-06-24 13:59:46 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/recorder/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /config/custom_components/bhyve/translations/switch.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_13) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/141897999.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/circle-sensor-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/api/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/mobile_app/translations/sensor.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_19) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/141952963.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/state-attribute-element, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /config/custom_components/bhyve/translations/sensor.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/met/translations/weather.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/honeywell/translations/climate.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ipp/translations/sensor.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/nx584/translations/alarm_control_panel.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_17) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/142038085.hacs2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/updater/translations/binary_sensor.en.json2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=ofekashery/vertical-stack-in-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/142051833.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/state-element, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_10) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/142344403.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/canvas-gauge-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/webhook/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/logbook/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/142545838.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=herikw/home-assistant-custom-components, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ffmpeg/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/template/translations/switch.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_14) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/143340728.hacs2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/frontend/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/history/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/cloud/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/system_log/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/smtp/translations/notify.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/http/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/ring/translations/binary_sensor.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_7) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/143647651.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-components/combined, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/device_automation/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/myq/translations/cover.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_18) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/143741845.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/dual-gauge-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /config/custom_components/bhyve/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/smartthings/translations/climate.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/map/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/homeassistant/translations/scene.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_13) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/143762825.hacs2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/mobile_app/translations/binary_sensor.en.json2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/beer-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/smartthings/translations/switch.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_19) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/143850865.hacs2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/search/translations/en.json2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=Paul-dH/Home-Assisant-Sensor-OvApi, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_11) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/144888844.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=thomasloven/lovelace-slider-entity-row, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_6) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/144899700.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/ext-weblink, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/group/translations/light.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/default_config/translations/en.json2020-06-24 13:59:47 DEBUG (SyncWorker_8) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/144902010.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=iantrich/aftership-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_12) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/145142810.hacs2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/smartthings/translations/cover.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/logger/translations/en.json2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/username-element, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_0) [homeassistant.util.json] JSON file not found: /usr/src/homeassistant/homeassistant/components/myq/translations/binary_sensor.en.json2020-06-24 13:59:47 DEBUG (SyncWorker_14) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/145143666.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-components/feedparser, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_9) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/145180996.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-components/sensor.personalcapital, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_1) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/145777833.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/button-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_13) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/146194325.hacs2020-06-24 13:59:47 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event hacs/repository[L]: id=1337, action=registration, repository=custom-cards/rmv-card, repository_id=0>2020-06-24 13:59:47 DEBUG (SyncWorker_15) [homeassistant.util.json] JSON file not found: /config/.storage/hacs/146335411.hacs

**Comment by sebr** [comment]
I've fixed this issue with version [`2.0.0`](https://github.com/sebr/bhyve-home-assistant/releases/tag/2.0.0)

Note that to fix this issue I needed to introduce breaking changes so the entities will be recreated. I suggest deleting the old ones and renaming the `*_2` entities.

I have also changed the State sensor logic to create only one sensor per device - rather than one per zone. It was incorrect to create a state sensor per zone.
